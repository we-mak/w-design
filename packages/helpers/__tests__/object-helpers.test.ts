import { randomProperty, getValFromObjKey } from "../";

const testObject = {
  foo: "lorem",
  bar: "ipsum",
  baz: "dolor"
};

describe("object", () => {
  // only need to check radom value picked up from object
  test("randomProperty", () => {
    expect(randomProperty(testObject)).toMatchSnapshot();
  });

  test("getValFromObjKey", () => {
    expect(getValFromObjKey("bar", testObject)).toBe("ipsum");
  });
});
