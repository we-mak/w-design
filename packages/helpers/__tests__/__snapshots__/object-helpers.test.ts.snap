// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`object randomProperty 1`] = `"lorem"`;
